lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module

```diff
@@ -746,31 +746,28 @@ function dosomething_helpers_get_user_info($drupal_id) {
     return NULL;
   }
 
-  $northstar_response = dosomething_northstar_get_northstar_user($drupal_id);
-  $northstar_response = json_decode($northstar_response);
-
-  if ($northstar_response && isset($northstar_response->error)) {
-    return NULL;
+  $northstar_user = dosomething_northstar_get_user($drupal_id);
+
+  if ($northstar_user) {
+    return [
+      'id' => data_get($northstar_user, 'id'),
```

Comment:
Yeah, I think I can do that, but need to test. There's other things I still need to update, but this first PR was just meant to tackle the `dosomething_northstar_get_northstar_user()` function and replace it w/ NS-PHP call. This is just the first step.